src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -204,8 +206,8 @@ public Map<String, String> getRecordFromServiceNowTable(String tableName, String
     requestBuilder.setAuthHeader(accessToken);
     apiResponse = executeGet(requestBuilder.build());
     JsonObject jsonObject = gson.fromJson(apiResponse.getResponseBody(), JsonObject.class);
-    JsonArray jsonResultArray = jsonObject.getAsJsonArray(ServiceNowConstants.RESULT);
-    Map<String, String> responseMap = gson.fromJson(jsonResultArray.get(0), Map.class);
+    serviceNowJsonResultArray = jsonObject.getAsJsonArray(ServiceNowConstants.RESULT);
```

Comment:
why did we need this change?

Reply:
ServiceNow response is being stored in this variable, so using this in the validation logic.